components/HelpScreen.tsx

```diff
@@ -31,39 +57,102 @@ export const HelpScreen: React.FC<HelpScreenProps> = props => {
         <ScrollView>
           <Column fill padding="10" align="space-between">
             <Text margin="7" style={Theme.TextStyles.header}>
-              {t('whatIsDigitalCredential?')}
+              {t('whatIsAnId?')}
+            </Text>
+            <Text style={Theme.TextStyles.helpDetails}>{t('detail-10')}</Text>
+            <Text margin="7" style={Theme.TextStyles.header}>
+              {t('whatAreTheDifferentTypesOfId?')}
+            </Text>
+            <Text style={Theme.TextStyles.helpDetails}>
+              {t('detail-11')}
+              {hyperLinkString(
+                'here',
+                diffTypesOfIdsUrl,
+                '1.2.0/id-lifecycle-management/identifiers',
```

Comment:
Where is it redirecting?

Reply:
Now I changed this name to "**customisedUrl**". It navigates to INJI life-cycle managements  [identifiers](https://docs.mosip.io/1.2.0/id-lifecycle-management/identifiers)